src/fu-debug.c

```diff
@@ -116,6 +118,9 @@ fu_debug_pre_parse_hook (GOptionContext *context,
 		{ "plugin-verbose", '\0', 0, G_OPTION_ARG_STRING_ARRAY, &self->plugin_verbose,
 		  /* TRANSLATORS: this is for plugin development */
 		  N_("Show plugin verbose information"), "PLUGIN-NAME" },
+		{ "daemon-verbose", '\0', 0, G_OPTION_ARG_STRING_ARRAY, &self->daemon_verbose,
+		  /* TRANSLATORS: this is for daemon development */
+		  N_("Show daemon verbose information"), "DOMAIN" },
```

Comment:
Maybe:
`Show daemon verbose information for individual domains`

Reply:
Nahh, domain is a term overloaded by multiple things. It needs to be translatable.

Reply:
OK